src/algorithms/masked_gibbs.jl

```diff
@@ -510,3 +510,49 @@ function assert_spikes_not_in_mask(
         end
     end
 end
+
+function clean_masks(
```

Comment:
What do you think about changing the name from `clean_masks` to `merge_contiguous_masks`? I think it might be a slightly more informative name. This is a nitpick -- I think we can merge this soon!

Reply:
Yeah, that makes sense, merge_continuous_masks is a much more helpful name!